GoogleDataTransport/GDTCORLibrary/GDTCOREndpoints.m

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google
```

Comment:
Please add `LLC` here and at other places.

Reply:
Done.